### PR TITLE
[ADD] udes_stock: Adding a config for current quantity on wharehouse

### PR DESCRIPTION
**Pull request by armandcela**
Story: 3515
Signed-off-by: Armand Cela <armand.cela@unipart.io>

### CODE REVIEW

**Review by kdwyer — APPROVED** [review]

**Comment by jrbirch99** [comment]
Q: why do we set this attribute at the warehouse level instead of the location level? It feels like (from reading the story) that it should be configurable per location

**Comment by jrbirch99** [comment]
Q: Should this be in the hds repo? On the story they seem to only want this functionality for hds :slightly_smiling_face:

**Comment by armandcela** [comment]
> Q: why do we set this attribute at the warehouse level instead of the location level? It feels like (from reading the story) that it should be configurable per location not just per warehouse

I think reasons are as below:
1. In story doesnt say that some location should have and some not to set it per location.
2. In story says needed only for HDS, thats why need smth to distinguish per client level , which warehouse distinguishes 
3. We may even scan package on lookup which shows products with quantities and we need to show current quantity in that case too.
4. Setting per location means that on every location need to set True or False which doesnt make sense i think

**Comment by jrbirch99** [comment]
> > Q: why do we set this attribute at the warehouse level instead of the location level? It feels like (from reading the story) that it should be configurable per location not just per warehouse
> 
> I think reasons are as below:
> 
>     1. In story doesnt say that some location should have and some not, to set it per location.
> 
>     2. In story says needed only for the customer, thats why need smth to distinguish per client level , which warehouse distinguishes
> 
>     3. We may even scan package on lookup which shows products with quantities and we need to show current quantity in that case too.
> 
>     4. Setting per location means that on every location need to set True or False which doesnt make sense i think

For point 4. You could have it so that if it was set to `True` only a parent location it propagated it to its child locations, which would mean for 2. we could just set it in data on say the "stock" location for HDS. But I think it is fine, as the flag is only used for showing things in the frontend, and we do that per picking type anyway which implies some filtering by location :slightly_smiling_face:

**Comment by adamrpatrick** [comment]
Typo: "Warehouse" instead of "Wharehouse"

File: addons/udes_stock/views/stock_warehouse.xml [hunk 1]
```diff
@@ -54,6 +54,7 @@
                             <field name="u_inventory_adjust_reserved" />
                             <field name="u_allow_create_picking_reserved_package" />
                             <field name="u_log_batch_picking" />
+                            <field name="u_show_current_quantity" />
```
Comment (adamrpatrick) [inline addons/udes_stock/views/stock_warehouse.xml:57]: Let's move this into the customer module.

**Review by jrbirch99 — APPROVED** [review]

**Pull request merged**